daemon/inertiad/build/builder.go

```diff
@@ -243,16 +247,25 @@ func (b *Builder) dockerBuild(d Config, cli *docker.Client,
 	buildResp.Body.Close()
 	fmt.Fprintf(out, "%s (%s) build has exited\n", imageName, buildResp.OSType)
 
+	// Get image details
+	image, _, err := cli.ImageInspectWithRaw(ctx, imageName)
+	if err != nil {
+		return nil, err
+	}
+	portMap := nat.PortMap{}
+	for p := range image.Config.ExposedPorts {
+		portMap[p] = []nat.PortBinding{{HostIP: "0.0.0.0", HostPort: p.Port()}}
```

Comment:
`{{...}}` wot. Is hash parameters not a thing in Go?

Reply:
What do you mean?

The outer `{}` is the array instantiator, the inner `{}` is for an element